config/config.exs

```diff
@@ -29,3 +29,6 @@ config :we_need_to_watch, :basic_auth,
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{Mix.env}.exs"
+
+# Override with local configuration (gitignored)
+import_config "local.exs"
```

Comment:
I think it'd like it to be conditional... use it if you want, don't if you don't
```
if File.exists?("./local.exs"), do: import_config "local.exs"
```

Thoughts?

Reply:
I'm fine with that. Should I also pull out the hard-coded `raelyn` user in `config/config.exs` and replace it with the traditional `postgres` user?

Reply:
I don't see that in `config/config.exs`... We should remove it from `config/dev.exs` if that is the one you meant.